docusaurus.config.js

```diff
@@ -57,6 +57,8 @@ const config = {
           blogTitle: "Blogs",
           blogDescription: "TCET Open Source Blogging Page",
           blogSidebarCount: 'ALL',
+          postsPerPage: 3,
```

Comment:
we might want to revise this in the future, as I believe we have around 10 entries on the sidebar